### PR TITLE
Recommend to use PHPUnitBridge instead of PHPUnit

### LINKED ISSUE #8808: Recommend installing phpunit bridge instead of phpunit
See https://github.com/symfony/recipes/pull/292#issuecomment-348570723

I added the 4.0 milestone, because Flex will show a warning. But really, this should likely be done in 3.3.

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #8808.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Review by weaverryan — CHANGES_REQUESTED** [review]
Should we use the `--dev` flag for all of the `compose require phpunit` situations?

File: setup/bundles.rst [hunk 1]
```diff
@@ -69,7 +69,8 @@ PHPUnit test report:
 
 .. code-block:: terminal
 
-    $ phpunit
+    # this command is available after running "composer require phpunit"
+    $ ./bin/phpunit
```
Comment (weaverryan) [inline setup/bundles.rst]: This is a third-party bundle, so I think we should use the full component name, not the alias, right?

Comment (xabbuh) [reply setup/bundles.rst]: The alias is for the bridge. So that looks good to me.

**Comment by javiereguiluz** [comment]
Ready for the final review. Thanks!

**Pull request closed without merging**